test/passes/generate-stack-ir_optimize-stack-ir_print-stack-ir_optimize-level=3.txt

```diff
@@ -20,15 +20,15 @@
  (func $big_negative
   (local $temp f64)
   f64.const -2147483648
-  local.set $temp
+  local.set $0
```

Comment:
Why did local names change?

Reply:
Hmmm, I'm not sure. That does seem fishy.